server/user/user_server/server_test.go

```diff
@@ -0,0 +1,72 @@
+package user_server
+
+import (
+	"reflect"
+	"io/ioutil"
+	"os"
+	"context"
+	"log"
+	"primitivofr/owly/server/user/userpb"
+	"testing"
+
+	"google.golang.org/grpc/metadata"
+)
+
+// "context"
+// "log"
+// "os"
+// "primitivofr/owly/user/auth/userpb"
+// "reflect"
+// "testing"
+
+// "google.golang.org/grpc/codes"
+// "google.golang.org/grpc/status"
+
+func TestSearchUserByUsername(t *testing.T) {
+	s := server{}
+
+	tests := []struct {
+		req  userpb.SearchUserByUsernameRequest
+		want userpb.SearchUserByUsernameResponse
+	}{
+		{
+			req: userpb.SearchUserByUsernameRequest{
+				Username: "user",
+			},
+			want: userpb.SearchUserByUsernameResponse{
+				Count: 1,
+			},
+		},
+		{
+			req: userpb.SearchUserByUsernameRequest{
+				Username: "notexist",
+			},
+			want: userpb.SearchUserByUsernameResponse{
+				Count: 0,
```

Comment:
Actually, RPC doesn't send back 0, null or "" values if they are not set as required in the protofile.

So something like Count: 0 won't happen. Maybe remove it for a better clarity